packages/docusaurus-1.x/lib/static/css/main.css

```diff
@@ -1975,6 +1975,10 @@ input::placeholder {
   color: #717171;
 }
 
+.onPageNav .toc-headings>li>a.active {
+    font-weight: 700!important;
```

Comment:
Why is `!important` needed? Adding `!important` makes it hard to override by users who don't want this behavior.

Reply:
Use 600 as it's the more common value in the code.